src/Web/Sfa.Das.Sas.Web.UnitTests/App.config

```diff
@@ -6,6 +6,11 @@
 <configuration>
   <appSettings>
     <add key="FeatureToggle.RoatpProviderFlagsFeature" value="true" />
+    <add key="DefaultResultsAmount" value="20" />
```

Comment:
Do we need this config in test project?

Reply:
On the error message, Rajesh said that standards should mean there's just one message, like the screenshot you posted shows.  I figured I'd leave it to testing/PO Review for Rajesh to explain that.

And no, that think isn't needed in the test.  I just put it in to figure out some testing, and forgot to remove it.